satpy/readers/ahi_l1b_gridded_bin.py

```diff
@@ -143,14 +141,13 @@ def _calibrate(self, data):
 
     @staticmethod
     def _download_luts(file_name):
-        """Download LUTs from remote FTP server."""
-        from ftplib import FTP
-        # Set up an FTP connection (anonymous) and download
-        ftp = FTP(AHI_REMOTE_LUTS[0])
-        ftp.login('anonymous', 'anonymous')
-        ftp.cwd(AHI_REMOTE_LUTS[1])
-        with open(file_name, 'wb') as _fp:
-            ftp.retrbinary("RETR " + AHI_REMOTE_LUTS[2], _fp.write)
+        """Download LUTs from remote server."""
+        import requests
```

Comment:
`requests` is not part of the stdlib, so it would have to be added to setup.py. However, an alternative is to use `urllib.request.urlopen` a few lines down instead of `requests.get`, but do you know if there is an advantage to using the latter over the former?

Reply:
Because I saw a S.O. post that said `don't use urllib`. ;-)
But it should be OK here. Will make the switch.

Reply:
Done!

Reply:
Thanks